mds/migrations/0039_map_old_to_new_event_records.py

```diff
@@ -98,9 +98,9 @@ def fill(qs):
 
 class Migration(migrations.Migration):
     atomic = False
-    deploy_phase = "0039_eventrecord_event_type_reason"
+    deploy_phase = "post_deploy"
```

Comment:
I didn't notice it in my initial review, I don't know if it will be taken into account.

That could be interesting, I'm sick of maintaining the external `migration_tags.yml` file.

If it does work, just put the deploy phase everywhere.

Reply:
So no it doesn't work. I'm going to remove it